redash/cli/rq.py

```diff
@@ -19,9 +21,27 @@ def scheduler():
     rq_scheduler.run()
 
 
+@manager.command()
+@argument('workers_count')
+@argument('queues', nargs=-1)
+def workers(workers_count=1, queues='default'):
+    m = Manager()
+    for i in range(int(workers_count)):
+        cmd = './manage.py rq worker {}'.format(" ".join(queues))
+        m.add_process('worker #{}'.format(i+1), cmd)
+
+    m.loop()
+    sys.exit(m.returncode)
+
+
 @manager.command()
 @argument('queues', nargs=-1)
 def worker(queues='default'):
+    # Configure any SQLAlchemy mappers loaded until now so that the mapping configuration 
+    # will already be available to the forked work horses and they won't need 
+    # to spend valuable time re-doing that on every fork.
+    configure_mappers()
```

Comment:
We should probably move this to its own PR, so it's not forgotten in case we abandon this one.

Reply:
🙊https://github.com/getredash/redash/pull/4314

Reply:
Oops :-)